### ISSUE TITLE
Phalcon\Acl\Resource not implementing Phalcon\Acl\ResourceInterface

### ISSUE THREAD

**Issue by rosasurfer** [phalcon/cphalcon#11959]
While the methods defined by the interface are in place the interface itself is not implemented. This causes an error if a Resource instance is explicitely tested against the interface:

```
use Phalcon\Acl\Resource;
use Phalcon\Acl\ResourceInterface;

$object = new Resource('id');

if ($object instanceof ResourceInterface) {
   echo 'instance of ResourceInterface';
}
else {
   echo 'not an instance of ResourceInterface';
}
```

Phalcon version 2.0.13
Please correct me if I'm wrong.

**Comment by Jurigag** [comment]
Well you are right - https://github.com/phalcon/cphalcon/blob/master/phalcon/acl/resource.zep#L29

**Comment by sergeyklay** [comment]
Fixed in the `3.0.x` branch.
